addons/main/config.cpp

```diff
@@ -8,6 +8,7 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {
             "A3_Data_F_Tank_Loadorder",
+            "A3_Data_F_Mod_Loadorder",
```

Comment:
Does this one not require `A3_Data_F_Tank_Loadorder`?

Reply:
it seems to just be itself, unless I'm reading it wrong

Reply:
https://cdn.discordapp.com/attachments/441242389823225876/519888830179049483/1.JPG

Fail